scripts/benchmarks/benches/compute.py

```diff
@@ -71,17 +71,20 @@ def benchmarks(self) -> list[Benchmark]:
 
         if options.ur is not None:
             benches += [
-                SubmitKernelUR(self, 0),
-                SubmitKernelUR(self, 1),
+                SubmitKernelURWithTime(self, 0),
+                SubmitKernelURWithTime(self, 1),
+                SubmitKernelURWithCPUCounter(self, 0),
+                SubmitKernelURWithCPUCounter(self, 1),
             ]
 
         return benches
 
 class ComputeBenchmark(Benchmark):
-    def __init__(self, bench, name, test):
+    def __init__(self, bench, name, test, unit="time [us]"):
```

Comment:
this looks very odd, please do an "enum" ([example](https://github.com/oneapi-src/unified-runtime/blob/main/scripts/benchmarks/benches/options.py#L4))